nextcloud_update.sh

```diff
@@ -308,15 +331,18 @@ https://shop.hanssonit.se/product/upgrade-between-major-owncloud-nextcloud-versi
     exit 1
 fi
 
-# Check if new version is larger than current version installed.
-print_text_in_color "$ICyan" "Checking for new Nextcloud version..."
-if version_gt "$NCVERSION" "$CURRENTVERSION"
+# Check if new version is larger than current version installed. Skip version check if you want to upgrade to a prerelease.
+if [ -z "$PRERELEASE_VERSION" ]
 then
-    print_text_in_color "$ICyan" "Latest release is: $NCVERSION. Current version is: $CURRENTVERSION."
-    print_text_in_color "$IGreen" "New version available, upgrade continues!"
-else
-    print_text_in_color "$IGreen" "You already run the latest version! ($NCVERSION)"
-    exit 0
+    print_text_in_color "$ICyan" "Checking for new Nextcloud version..."
+    if version_gt "$NCVERSION" "$CURRENTVERSION"
+    then
+        print_text_in_color "$ICyan" "Latest release is: $NCVERSION. Current version is: $CURRENTVERSION."
+        print_text_in_color "$IGreen" "New version available, upgrade continues!"
+    else
+        print_text_in_color "$IGreen" "You already run the latest version! ($CURRENTVERSION)"
```

Comment:
@enoch85 I changed this report to report the latest installed version.

Reply:
Nice catch!